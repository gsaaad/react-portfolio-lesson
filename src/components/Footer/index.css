footer {
  font-size: 30px;
  height: 50px;
  border-top: 5px solid #00aeff;
  background-image: linear-gradient(black, #00aeff);
}
