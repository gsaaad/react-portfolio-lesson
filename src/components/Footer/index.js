import React from "react";
import "./index.css";
function Footer() {
  return <footer>ğŸ’¡Made with ReactğŸ’¡</footer>;
}

export default Footer;
