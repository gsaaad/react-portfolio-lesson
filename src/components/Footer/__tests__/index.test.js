import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "..";

// go through each test and after, clean up and dispose of anything we aren't using
afterEach(cleanup);

describe("Footer component", () => {
  // render footer test

  // first test
  it("renders", () => {
    render(<Footer />);
  });
});

describe("Matches snapshot DOM node structure", () => {
  // use asFragment to return snapshot of DOM

  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
