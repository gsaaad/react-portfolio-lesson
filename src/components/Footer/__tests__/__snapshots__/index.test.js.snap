// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <footer>
    ðŸ’¡Made with ReactðŸ’¡
  </footer>
</DocumentFragment>
`;
